From 3752337216c795fa5e6627d751f50d4ef52ed70a Mon Sep 17 00:00:00 2001
From: zachary zhang <zhangzg@marvell.com>
Date: Tue, 2 Jan 2018 18:42:26 +0800
Subject: [PATCH 0662/1200] arm64: dts: a70x0: a80x0: update board model's name
 to align with PCB printing

Currently, a7k/8k dts board name is very brief.
This patch update dts model's name to align with PCB printing information.

Change-Id: I6d16a1438c30b5274e75c9edcaeac4e2d2450daf
Signed-off-by: zachary zhang <zhangzg@marvell.com>
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/55228
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
---
 arch/arm/dts/armada-7040-db-nand.dts | 2 +-
 arch/arm/dts/armada-7040-db.dts      | 2 +-
 arch/arm/dts/armada-8040-db.dts      | 2 +-
 arch/arm/dts/armada-8040-mcbin.dts   | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/dts/armada-7040-db-nand.dts b/arch/arm/dts/armada-7040-db-nand.dts
index e3a5a715c0..2ca85a4f6d 100644
--- a/arch/arm/dts/armada-7040-db-nand.dts
+++ b/arch/arm/dts/armada-7040-db-nand.dts
@@ -48,7 +48,7 @@
 #include "armada-7040.dtsi"
 
 / {
-	model = "Marvell Armada 7040 DB board with NAND";
+	model = "Marvell Armada 7040 Development Board DB-88FA7040 with NAND";
 	compatible = "marvell,armada7040-db-nand", "marvell,armada7040-db",
 		     "marvell,armada7040", "marvell,armada-ap806-quad",
 		     "marvell,armada-ap806";
diff --git a/arch/arm/dts/armada-7040-db.dts b/arch/arm/dts/armada-7040-db.dts
index 1ed23ecc55..e095f42797 100644
--- a/arch/arm/dts/armada-7040-db.dts
+++ b/arch/arm/dts/armada-7040-db.dts
@@ -48,7 +48,7 @@
 #include "armada-7040.dtsi"
 
 / {
-	model = "Marvell Armada 7040 DB board";
+	model = "Marvell Armada 7040 Development Board DB-88FA7040";
 	compatible = "marvell,armada7040-db", "marvell,armada7040",
 		     "marvell,armada-ap806-quad", "marvell,armada-ap806";
 
diff --git a/arch/arm/dts/armada-8040-db.dts b/arch/arm/dts/armada-8040-db.dts
index 35bddca06e..5f7fb5b4c4 100644
--- a/arch/arm/dts/armada-8040-db.dts
+++ b/arch/arm/dts/armada-8040-db.dts
@@ -47,7 +47,7 @@
 #include "armada-8040.dtsi"
 
 / {
-	model = "Marvell Armada 8040 DB board";
+	model = "Marvell Armada 8040 Development Board DB-88F8040";
 	compatible = "marvell,armada8040-db", "marvell,armada8040",
 		     "marvell,armada-ap806-quad", "marvell,armada-ap806";
 
diff --git a/arch/arm/dts/armada-8040-mcbin.dts b/arch/arm/dts/armada-8040-mcbin.dts
index 880e76d02c..df41471784 100644
--- a/arch/arm/dts/armada-8040-mcbin.dts
+++ b/arch/arm/dts/armada-8040-mcbin.dts
@@ -8,7 +8,7 @@
 #include "armada-8040.dtsi" /* include SoC device tree */
 
 / {
-	model = "MACCHIATOBin-8040";
+	model = "Marvell Armada 8040 MACHIATOBin";
 	compatible = "marvell,armada8040-mcbin",
 		     "marvell,armada8040";
 
-- 
2.22.0

